sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_6_day_offset_v1/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+description: >
+  Measures the number of activated new profiles in the Firefox desktop
+  funnel. The acitvation date is offset by 6 days to reduce complexity 
+  for scheduling in Airflow. For querying desktop funnel activation data, 
+  use telemetry_derived.desktop_funnel_activation_v1 instead.
+friendly_name: Firefox Desktop Funnel Activation 6 Day Offset
```

Comment:
Naming is hard. I'm open to better suggestions for this query.

Reply:
This seems roughly in line with some existing terminology for the `smoot_usage_day_6` view, etc. Those tables for GUD rely heavily on looking at bit patterns at different offsets for forward-looking windows. I think it's good to codify the offset in the table name here for least surprise.